client/src/js/subtraction/components/Selector.js

```diff
@@ -2,14 +2,24 @@ import { map } from "lodash-es";
 import PropTypes from "prop-types";
 import React from "react";
 import { MultiSelector, MultiSelectorItem } from "../../base/MultiSelector";
+import { Link, BrowserRouter } from "react-router-dom";
+import { NoneFoundBox } from "../../base/NoneFound";
 
 export const SampleSubtractionSelector = ({ name, noun, selected, subtractions, onChange }) => {
     const subtractionComponents = map(subtractions, ({ name, id }) => (
         <MultiSelectorItem key={id} name={name} value={id} id={id}>
             <span>{name}</span>
         </MultiSelectorItem>
     ));
-
+    if (!subtractions.length) {
+        return (
+            <NoneFoundBox noun={"subtractions"}>
+                <BrowserRouter>
```

Comment:
Why did you need `BrowserRouter` here?

Reply:
Turns out I don't need it there. Jest was complaining that the Link wasn't wrapped while running test because it lacked the normal context of the router from the parent components. Fixed in latest version by wrapping the whole component in the test rather than adding a new router to the production code.